objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -369,7 +399,7 @@ public void run() {
             source.pollTimer.schedule(task, 0, pollRate);
         } else if (polling == 0) {
             source.pollTimer = new Timer("dataCapture");
-            source.pollTimer.scheduleAtFixedRate(task, 0, 1);
+            source.pollTimer.schedule(task, 0, 1);
```

Comment:
What are we doing here?  (Need a comment in the code rather than answering this...)

Is this doing constant/immediate polling?

Reply:
I will add a comment in the code as well, but figured I'd explain here too. This change means that instead of scheduling all tasks at a fixed rate from the first task, we will schedule them relative to the previous execution.

This means that if there are any delays while executing a task, the next task will still be scheduled `pollTime` milliseconds after the previous task has completed.